### PR TITLE
[WIP] - Notifications integration

### PR DESCRIPTION
**Pull request by jontours**
## Description
This is a possible integration for adding notifications to Tock.  Some integration work still to go.  In fact, perfectly ok for this feature to never see the light of day, but to maybe get the conservation going.

This work was to demonstrate feasibility for the Notifications-as-a-Service 10X.

I'm not a Django programmer but this was the simplest way to get a simple webform started with limited impact.

image

### CODE REVIEW

File: tock/notifications/views.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+from django.views.generic.edit import FormView
+
+def index(request):
+    return HttpResponse("Hello, world. You're at the polls index.")
+
+class Notification():
+    notification_label = "Friday alert"
+    notification_text = "Tock your Time!"
+
+class NotificationsListView(DetailView):
```
Comment (tbaxter-18f) [inline tock/notifications/views.py]: Just curious: Is there a reason to call it ListView but subclass DetailView?

**Comment by tbaxter-18f** [comment]
I think there's the start of a good idea here, but I have two extra thoughts:

1. Could we leverage Django's built-in messages framework for the on-site messages themselves?
2.  Is this a rare good use of push notifications?

**Comment by tbaxter-18f** [comment]
I'm relating this to #653 so we can take a holistic approach to notifications.

**Comment by jontours** [comment]
This work has been discussed a bit lately.  But more in the context of maybe utilizing it as a cloud.gov broker.  The UK GDS has rolled out a shared notifications service that is mostly similar to the ideas @amoose and I threw around.

**Comment by tbaxter-18f** [comment]
I'm definitely intrigued, and when I -- or whoever -- picks this up again, let's definitely meet up and walk through all these options.

**Comment by tbaxter-18f** [comment]
Given the age of this and the work @mgwalker has been doing with Slack integration and notifications, I'm going to go ahead and close this.

**Pull request closed without merging**